nevergrad/optimization/test_optimizerlib.py

```diff
@@ -432,9 +432,10 @@ def test_bo_parametrization_and_parameters() -> None:
     parametrization = ng.p.Instrumentation(ng.p.Choice([True, False]))
     with pytest.warns(errors.InefficientSettingsWarning):
         xpvariants.QRBO(parametrization, budget=10)
-    with pytest.warns(None) as record:  # type: ignore
+    with pytest.warns(None) as record:
```

Comment:
mypy is wrong here so ignore it

```suggestion
    with pytest.warns(None) as record:  # type: ignore
```